cmd/podman/system_df.go

```diff
@@ -501,7 +501,7 @@ func getContainerVerboseDiskUsage(containers []*libpod.Container) (containersVer
 			Command:      strings.Join(ctr.Command(), " "),
 			LocalVolumes: len(ctr.UserVolumes()),
 			Size:         units.HumanSizeWithPrecision(float64(size), 3),
-			Created:      units.HumanDuration(time.Since(ctr.CreatedTime().Local())) + "ago",
+			Created:      units.HumanDuration(time.Since(ctr.CreatedTime().Local())) + " ago",
```

Comment:
could we rewrite it using fmt.Sprintf()?

`fmt.Sprintf("%s ago", units.HumanDuration(time.Since(ctr.CreatedTime().Local())))`